common/src/abstractions/organization.service.ts

```diff
@@ -0,0 +1,11 @@
+import { OrganizationData } from '../models/data/organizationData';
+
+import { Organization } from '../models/domain/organization';
+
+export abstract class OrganizationService {
+    get: (id: string) => Promise<Organization>;
+    getByIdentifier: (identifier: string) => Promise<Organization>;
+    getAll: (userId?: string) => Promise<Organization[]>;
+    save: (orgs: {[id: string]: OrganizationData}) => Promise<any>;
+    canManageSponsorships: () => Promise<boolean>;
```

Comment:
This method feels slightly out of place. I understand it was just moved from `userService`, and I don't mind letting it through but I wonder if we could just move the logic to the function and have something like `orgService.getAll().any(o => o.canManageSponsorship)` in `settings.component.ts`. I'm fine if we add it as a tech dept or something.

Reply:
https://app.asana.com/0/1201461753012676/1201507368222862